README.md

```diff
@@ -117,3 +117,27 @@ You can also specify a `TESTARGS` variable to filter tests like so:
 ```sh
 $ make test TESTARGS='--run=TestConfig'
 ```
+
+#### Acceptance Tests
+
+**Warning:** The acceptance tests create/destroy/modify *real resources*,
+which may incur real costs in some cases. In the presence of a bug,
+it is technically possible that broken backends could leave dangling
+data behind. Therefore, please run the acceptance tests at your own risk.
+At the very least, we recommend running them in their own private
+account for whatever backend you're testing.
```

Comment:
The workflow does not have a clean-up step, does it mean each run will leak a few resources that accumulate over time?

Reply:
No, the acceptance tests are not leaking resources. The tests are using an existing role and then creating temporary credentials to login. This text was copied from another plugin. I can reword it if you think it is worthwhile?

Reply:
Nah its fine, I was mostly worried the [acc-tests.yaml](https://github.com/hashicorp/vault-plugin-auth-alicloud/pull/67/files#diff-d8911d58018995b0076267be89bd3e997bce15cfbe4f28e0f5c37d032ba7befa) workflow would pile up costs in our account but from what I understand this particular plugin's acc tests do not create paid resources.